components/core/src/utils/make_dictionaries_readable/make-dictionaries-readable.cpp

```diff
@@ -81,8 +86,15 @@ int main (int argc, const char* argv[]) {
 
         file_writer.write_string(replace_characters("\n", "n", human_readable_value, true));
         file_writer.write_char('\n');
+        std::set<segment_id_t> segment_ids_set = entry.get_ids_of_segments_containing_entry();
+        // the ids_set is std::set which iterates id in ascending order
```

Comment:
How about `// segment_Ids is a std::set, which iterates the IDs in ascending order`?